### DISCUSSION TITLE
Privileged container

### DISCUSSION THREAD

**Discussion by dean0999** [Floorp-Projects/Floorp#1174] [Q&A] [answered]
Hi all, 
To begin with, I'm a Linux noob and getting used to the system, so silly questions are enabled.  When I open Floorp, a privileged container opens at the same time. I thought it might be to do with my pw manager (running in Flatpak) needing access to the browser, but I'd rather be certain and hear from someone who knows. Thanks! 
![Floorp privileged container](https://github.com/Floorp-Projects/Floorp/assets/163382257/3ceb97d4-adf0-4304-bd37-3afcd6109786)

**Comment by CutterKnife** [comment, marked as answer]
The name has been changed due to localization, but it is probably the Private Container feature.
This is one of Floorp's unique features, and tabs using this container will not keep history, cookies, etc.

**Reply by dean0999** [reply]
Thanks for the fast reply Cutterknife, really appreciated.
